microsoftgraph/microsoft-graph-explorer-v4
# Fix: Update query field when a sample query is selected

## Overview
This fix removes the redundant setSampleQuery action that is dispatched whenever the editor has a value update e.g a POST or PATCH request...
The second action would reset the query field back to its previous value
![image](https://user-images.githubusercontent.com/18407044/161746960-e2ece528-79bd-4d35-ab47-9f53fb1bbbac.png)

Fixes #1601

### Demo

![image](https://user-images.githubusercontent.com/18407044/161747597-5ec138d1-4c77-4579-93ce-5929f2ce6f33.png)

## Testing Instructions

* Load this PR locally and run Graph Explorer
* Select a query from **Batching** or any other POST/PATCH query
* Observe fix
